CONTRIBUTORS.md

```
@@ -1,28 +1,10 @@
-1. Fork the repository
-2. Create your feature branch
-3. Commit your changes using [Conventional Commits](https://www.conventionalcommits.org/)
-4. Push to your branch
-5. Open a Pull Request
-
-For more information, please see our [Contributing Guidelines](CONTRIBUTING.md).
+## Contributors
+
+| Contributions | Name                        | Email                                      |
+|---------------|-----------------------------|--------------------------------------------|
+| 55            | Steve Morin                 | steve.morin@gmail.com                     |
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)